bookwyrm/templates/annual_summary/layout.html

```diff
@@ -224,7 +224,7 @@ <h2 class="title is-3 is-serif">
     <div class="columns">
         <div class="column has-text-centered">
             <h2 class="title is-3 is-serif">
-            {% blocktrans %}All the books {{ display_name }} read in 2021{% endblocktrans %}
```

Comment:
thank you! good catch

Reply:
ah @joachimesque beat you to the punk on this one with #1710 -- thank you both for catching this and fixing it so quickly